ckanext/iati_generator/templates/package/snippets/resource_form.html

```diff
@@ -0,0 +1,170 @@
+
+{% import 'macros/form.html' as form %}
```

Comment:
It's not required to fully override the resource form template
Read this in the CKAN docs: https://docs.ckan.org/en/2.9/extensions/adding-custom-fields.html#adding-custom-fields-to-resources

We can manually define new fields and the use this file only to override the `{% block basic_fields_url %}` or the `{% block metadata_fields %}` (below in the form page)